src/main/java/com/transferwise/openbanking/client/json/VrpConsentResponseStatusEnumDeserializer.java

```diff
@@ -0,0 +1,32 @@
+package com.transferwise.openbanking.client.json;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.transferwise.openbanking.client.api.payment.v3.model.vrp.OBDomesticVRPConsentResponseData.StatusEnum;
+import java.io.IOException;
+
+public class VrpConsentResponseStatusEnumDeserializer
+    extends StdDeserializer<StatusEnum> {
```

Comment:
Why this is on a new line?

Reply:
The line was getting bigger. So, I chopped it. 
I remember from our previous discussion that some things like chopping are not formatted by IntelliJ editor config. We do if lines looks bigger. 
What is the problem in this? 
Looks to me, if I don't chop these lines then you will have comment and if I chop then you will have comment. What should I do :) ?

Reply:
It less than 150 characters(our limit) so it is not necessary to move it to the next line